distributed-job-manager/src/main/java/ru/fix/distributed/job/manager/ScheduledJobExecution.java

```diff
@@ -56,16 +52,11 @@ public ScheduledJobExecution(
         this.workShare = workShare;
         this.profiler = profiler;
         this.lockManager = lockManager;
-        this.jobDisableConfig = jobDisableConfig;
         this.zkPathsManager = zkPathsManager;
     }
 
     @Override
     public void run() {
-        if (!jobDisableConfig.get().isJobShouldBeLaunched(job.getJobId().getId())) {
-            log.trace("Job {} wasn't launched due to jobDisableConfig", job.getJobId());
-            return;
-        }
```

Comment:
we can keep this code for faster reaction on changes in jobDisableConfig, but i really think it's better to make Manager the only entity responsible for reacting on changes in jobDisableConfig